rplugin/node/vim-package-info/diff.js

```diff
@@ -35,27 +35,36 @@ function colorizeDiff(current, latest, hl) {
       [".", hl],
       [l.patch, "VimPackageInfoMajor"],
     ];
-  } else if (c !== null && parseInt(l.minor) > parseInt(c.minor)) {
-    cd = [
-      ...cd,
-      [l.major, hl],
-      [".", hl],
-      [l.minor, "VimPackageInfoMinor"],
-      [".", hl],
-      [l.patch, "VimPackageInfoMinor"],
-    ];
-  } else if (c !== null && parseInt(l.patch) > parseInt(c.patch)) {
-    cd = [
-      ...cd,
-      [l.major, hl],
-      [".", hl],
-      [l.minor, hl],
-      [".", hl],
-      [l.patch, "VimPackageInfoPatch"],
-    ];
+  } else if (c !== null && parseInt(l.major === parseInt(c.major))) {
```

Comment:
The brackets seem to be misaligned.
```suggestion
  } else if (c !== null && parseInt(l.major) === parseInt(c.major)) {
```